extensions/open-graph/package.json

```diff
@@ -7,7 +7,8 @@
   "author": "davidho0403",
```

Comment:
The CI error is due to my GitHub username being changed from davidho0403 to 1weiho. Could you please help me fix it? Thank you.

Reply:
Done 👍

Reply:
Thanks🥹